plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -154,6 +164,21 @@ public boolean isRankActivated() {
     }
 
     /**
+     * Added to properly uncoercing.
+     */
+    public boolean getIsRankActivated() {
+        return isRankActivated;
```

Comment:
I think the data migration is a risky process, we can live with this method name (unless you are absolutely against it).